web/src/composebox_typeahead.js

```diff
@@ -1235,6 +1236,6 @@ export function initialize({on_enter_send}) {
     $("#private_message_recipient").on("blur", function () {
         const val = $(this).val();
         const recipients = typeahead_helper.get_cleaned_pm_recipients(val);
-        $(this).val(recipients.join(", "));
+        $(this).val(util.format_array_as_list(recipients, "long", "conjunction"));
     });
```

Comment:
Do you know what this block does? I wonder if it's stale code predating the input pill code; @evykassirer FYI in case you know. 

(In any case, my actual question is what this looks like, so a screenshot would be great)

Reply:
(I'm not very familiar with with the private message pill code yet, so I don't think I can help here without diving further into it)

Reply:
According to me, it is a stale code which is predating the input pill code. As we can see it's effect has to be shown here: 
![image](https://github.com/zulip/zulip/assets/91906953/4e81516e-a77f-44e5-afa3-1bd50cd8e86c)

But it is overridden by input pill code.

The usage of it can be seen in ```compose.hbs``` 

``` html 
<div id="compose-direct-recipient" class="pill-container" data-before="{{t 'You and' }}">
                                <div class="input" contenteditable="true" id="private_message_recipient" data-no-recipients-text="{{t 'Add one or more users' }}" data-some-recipients-text="{{t 'Add another user...' }}"></div>
                            </div>

Reply:
Yeah so probably the right answer is to delete that function instead of updating it, in its own commit with a nice commit message.

Reply:
Done.